src/components/ErrorAndLoading.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import PageLoading from "./PageLoading";
+
+export default function ErrorAndLoading({ error, isLoading }) {
+  return (
+    <section className="px-5">
+      {error && <h1 className="text-xl text-center py-20 md:text-md text-red-600">We apologize for the inconvenience, but we are currently experiencing a (500) Server Error. Our team is aware of the issue and is actively working to resolve it. Thank you for your patience.</h1>}
```

Comment:
Keep this message short and more global, in case you have a 401 or 409 error, I guess you don't want to tell the user that it's a 500 error